makefile

```diff
@@ -1,46 +1,57 @@
-.DEFAULT_GOAL := all
-.PHONY: all release clean
-TODAY ?=$(shell date +%Y-%m-%d)
-VERSION=v$(TODAY)
-SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
-
-
-# MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
-all: tmp/output/release/icd11foundation.owl
-
-clean:
-	rm -rf tmp/
-
-tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
-	robot remove --axioms equivalent -i $< -o $@
-
-# Cleans all unicode characters
-tmp/output/intermediate1-unicode-cleaned.owl: tmp/input/source.owl | tmp/output/release/
-	tr -cd '\11\12\15\40-\176' < $< > $@
-
-tmp/output/release/:
-	mkdir -p $@
-
-tmp/input/:
-	mkdir -p $@
-
-tmp/input/source.owl: tmp/input/source.gz
-	gunzip -c $< > $@
-
-tmp/input/source.gz: | tmp/input/
-	wget ${SOURCE_URL} -O $@
-
-# Requires GitHub CLI: https://cli.github.com/
-release: | tmp/output/release/
-	@test $(VERSION)
-	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
-
-# HELP -----------------------------------------------------------------------------------------------------------------
-help:
-	@echo "-----------------------------------"
-	@echo "	Command reference: GARD OWL Ingest"
-	@echo "-----------------------------------"
-	@echo "all"
-	@echo "Runs ingest and creates all release artefacts.\n"
-	@echo "release"
-	@echo "Uploads release to GitHub.\n"
+.DEFAULT_GOAL := all
+.PHONY: all release clean
+TODAY ?=$(shell date +%Y-%m-%d)
+VERSION=v$(TODAY)
+SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
+
+
+# MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
+all: tmp/output/release/icd11foundation.owl
+
+clean:
+	rm -rf tmp/
+
+tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
+	robot remove --axioms equivalent -i $< -o $@
+
+# Cleans all unicode characters
+tmp/output/intermediate1-unicode-cleaned.owl: tmp/input/source.owl | tmp/output/release/
+	tr -cd '\11\12\15\40-\176' < $< > $@
+
+tmp/output/release/:
+	mkdir -p $@
+
+tmp/input/:
+	mkdir -p $@
+
+tmp/input/source.owl: tmp/input/source.gz
+	gunzip -c $< > $@
+	rm $<
+
+tmp/input/source.gz: | tmp/input/
+	wget ${SOURCE_URL} -O $@
+
+# Requires GitHub CLI: https://cli.github.com/
+release: | tmp/output/release/
+	@test $(VERSION)
+	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
+
+# Mappings
+tmp/input/mappings.json: | tmp/input/
+	wget -qO tmp/input/en_product1.json.tar.gz https://www.orphadata.com/data/json/en_product1.json.tar.gz
+	tar -xvf tmp/input/en_product1.json.tar.gz -C tmp/input/
+	mv tmp/input/en_product1.json $@
+	rm tmp/input/en_product1.json.tar.gz
+
+tmp/output/release/ordo-icd11.sssom.tsv: tmp/input/mappings.json | tmp/output/release/

```

Comment:
## Strategy: Get _exact matches_ to _foundation terms_
Basically this entails intersecting findings in both of these threads:
- [How determine ICD11 foundation terms (not a linearization, like MMS)](https://github.com/monarch-initiative/icd11/pull/11/files#r1542135331)
- [How to determine exact matches](https://github.com/monarch-initiative/icd11/pull/11/files#r1542132321)

My current strategy is: Query the OWL to find ICD11 MMS exact matches. Then do a lookup on the nomenclature file to get the ICD11 foundation term.

Reply:
Your analysis looks ok, but @twhetzel should review all that; after just five minutes reading through your analysis I would suggest you just go ahead and create a ORDO-ICD11 foundation mapping in whatever way you prefer, and explain how you did it in our next meeting.